_data/student_projects.yml

```diff
@@ -17,6 +17,11 @@
   image: emberjs_0x48piraj.png
```

Comment:
@Ploypiti Please correct this project link also 👍

Reply:
Fixed.